HISTORY.md

```diff
@@ -51,6 +51,9 @@
 
   * Change neural network types to avoid unnecessary use of rvalue references
     (#2259).
+  
+  * Adding a warning message in ffn_impl.hpp, brnn_impl.hpp and rnn_impl.hpp
+    (#2238).
```

Comment:
To me it seems like this isn't a very descriptive message.  To be honest this is a really minor change so I'd be okay with removing this bit from the HISTORY file.  Alternately, if you want to keep it, please clarify it a bit so that it's actually clear what the change that users will experience is.  (Preferably keep it short too. :))